tensorflow/core/kernels/segment_reduction_ops.h

```diff
@@ -66,36 +86,70 @@ struct UnsortedSegmentFunctor {
 };
 
 #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+
+// atomic reduction functors for the gpu
```

Comment:
Please start with uppercase and end with period.

Reply:
Have done!